debian/control

```diff
@@ -0,0 +1,12 @@
+Source: opx-dhcp-agent
```

Comment:
Does this package ship the DHCP libs? If so, should they be in a separate package?
[debian policy](https://www.debian.org/doc/packaging-manuals/python-policy/module_packages.html)